FinancialTracker/ViewControllers/HomeViewController.swift

```diff
@@ -83,8 +85,45 @@ class HomeViewController: UIViewController {
         }
     }
     
+    @objc func sendMail() {
+        guard MFMailComposeViewController.canSendMail() else {
+            assertionFailure("Mail services are not available")
+            return
+        }
+        
+        let composer = MFMailComposeViewController()
+        composer.mailComposeDelegate = self
```

Comment:
Try to make the composer more complex - add title to to the email, add some preconfigured text for example. 

You could also add some preconfigured contact support options that fill in different preconfigured text for the email - check out `UIAlertController` with `ActionSheet` for that and see what you come up with. 

It is not anything super difficult but it is worth checking out what you can do with action sheets - it is another commonly used iOS functionality.